web/cobrands/sass/_base.scss

```diff
@@ -1496,6 +1497,48 @@ html.js #map .noscript {
       background-image:url($image-sprite);
       background-position: flip(right, -341px) -3936px;
     }
+    &.expand {
+      padding: 0.6em 1em 0.5em 1em;
+      &:after {
```

Comment:
Fair enough! Hmm, not sure the LTR/RTL code has had to deal with an after/before swap before, I'll leave that with you to try and sort ;) There are many uses of the left/right and flip() mixins that hopefully make sense. `.expand` and `.close` can share most of their CSS too.